cloud/endagaweb/views/user.py

```diff
@@ -103,11 +109,29 @@ def auth_and_login(request):
     user = authenticate(username=request.POST['email'],
                         password=request.POST['password'])
     if user:
-        login(request, user)
-        next_url = '/dashboard'
-        if 'next' in request.POST and request.POST['next']:
-            next_url = request.POST['next']
-        return redirect(next_url)
+        if user.is_active:
```

Comment:
You need to check `user` is not `None` first in the case `authenticate` returns `None`